internal/pkg/api/server/templates/dependencies/dependencies.go

```diff
@@ -160,3 +163,10 @@ func (v *container) StorageApiToken() (string, error) {
 	// The API is authorized separately in each request
 	return "", nil
 }
+
+func (v *container) Clone() *container {
+	clone := *v
+	clone.commonDeps = clone.commonDeps.Clone()
+	clone.commonDeps.Abstract = &clone
+	return &clone
+}
```

Comment:
So spravnou implementaciou clone to funguje.

Templates dependencies su totiz rozdelene na 2 casti, ktore su vzajomne prepojene:
- `dependencies.CommonContainer` - spolocny kod s CLI
- a `templates/dependencies.container` - unikatny kod pre API

A pri klonovani som tam omylom nechaval referenciu na povodny `CommonContainer`, nebol tam teda `ctx` request,  a teda po request `ctx` neskoncil, a repo sa neodomklo.